RobotsWantedLeague/Program.cs

```diff
@@ -1,9 +1,16 @@
+using RobotsWantedLeague.Models;
 using RobotsWantedLeague.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
+// database settings
+
+builder.Services.Configure<RobotsDatabaseSettings>(
```

Comment:
This is the important part of the config change.  This is what is needed to wire up mongodb as an ASP service.